targets/PhpSdk/make.js

```diff
@@ -27,13 +28,22 @@ function makeApi(api, sourceDir, apiOutputDir) {
         getCurlAuthParams: getCurlAuthParams,
         getCustomApiSignatures: getCustomApiSignatures,
         getRequestActions: getRequestActions,
-        sourceDir: sourceDir
+        sourceDir: sourceDir,
+        getVerticalNameDefault: getVerticalNameDefault
     };
 
     var apiTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/PlayFabApi.php.ejs"));
     writeFile(path.resolve(apiOutputDir, "PlayFab" + api.name + "Api.php"), apiTemplate(locals));
 }
 
+function getVerticalNameDefault() {
+    if (exports.verticalName) {
+        return "\"" + exports.verticalName + "\"";
+    }
+
+    return "null";
```

Comment:
null isn't correct in PHP.
I think it's all caps:
http://php.net/manual/en/language.types.null.php